next.config.js

```diff
@@ -1,7 +1,16 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {
+
+module.exports = {
   reactStrictMode: true,
   basePath: '/academia',
+  async redirects() {
+    return [
+      {
+        source: '/academia/courses',
+        destination: '/academia',
+        permanent: false,
+        basePath: false,
+      },
+    ];
+  },
```

Comment:
Good call here, I would double check with Elizabeth Baggan (if you haven't already), but I assume this would be fine.

Reply:
i proposed that on the SEO ticket yesterday and they said they were cool with it...was my only concern about introducing a subcategory with no landing page associated with it